iothub_client/tests/iothubtr_amqp_msgr_ut/iothubtr_amqp_msgr_ut.c

```diff
@@ -959,6 +959,79 @@ static AMQP_MESSENGER_HANDLE create_and_start_messenger2(AMQP_MESSENGER_CONFIG*
     return handle;
 }
 
+char* umock_stringify_AMQP_MESSENGER_MESSAGE_DISPOSITION_INFO(const AMQP_MESSENGER_MESSAGE_DISPOSITION_INFO* value)
+{
```

Comment:
Tye, do you know when is stringify used? is it just for logging?
Generating an empty string doesn't seem to be the correct way. What do you think?

Reply:
This is how another passing UT was set up for BINARY_DATA structure being umock registered, my understanding is when you register a VALUE_TYPE you need a stringify, are_equal, copy, and free, for umock to register custom types like structs as per (https://github.com/Azure/umock-c/blob/master/doc/umock_c.md) so stringify is used to return a string representation of a custom value type, I also concur that an empty string doesn't seem to be correct, however the function is just needed to register the value type for umock, it doesn't change anything else in the UT.

Reply:
https://github.com/Azure/azure-iot-sdk-c/blob/f5728ee3ddb39a76890fe1220fe286f557ed3eec/iothub_client/tests/iothubtransportamqp_methods_ut/iothubtransportamqp_methods_ut.c

Reply:
Will investigate with Dan.